wikipron.py

```diff
@@ -225,11 +227,33 @@ def _scrape(data, config: _Config):
             if " " in pron:
                 continue
             pron = config.process_pron(pron)
-            print(f"{word}\t{pron}", file=config.output)
+            yield (word, pron)
+
+
+def scrape(config: Config) -> Iterator[Pair]:
```

Comment:
Given this function signature, the API design looks like this:

```python
from wikipron import Config, scrape
config = Config(...)  # kwargs needed; they are the same as those for the CLI -- run "wikipron -h" for the details
scrape(config)  # yields the (word, pronunciation) pairs
```

Should we document this in the readme?

Reply:
Also, any thoughts on how we can write tests for this key function?

Reply:
Added documentation for this usage to the README.

Not sure how to test quickly, but it's no more or less tested than before...

Reply:
Thanks for adding the docs!

Now that we have this `scrape()` function nicely decoupled from the CLI (previously it'd probably be much clunkier to test), we probably should test it, to avoid surprises down the road. I could try to come up with something in a future PR, so that we don't hold off your current one here.

Reply:
Agreed.